staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder_test.go

```diff
@@ -781,7 +781,7 @@ func TestBuildOpenAPIV3(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			gotSchema := *got.Components.Schemas["io.k8s.bar.v1.Foo"]
+			gotSchema := *got.Components.Schemas["bar.k8s.io.v1.Foo"]
```

Comment:
this should move into the commit that causes this change

Reply:
Is it intentional?

Reply:
Yes I was in a discussion with @Jefftree about the reverse-DNS style definition naming and we decided not to retain it.

Reply:
@Jefftree after thinking about it some more I think we should restore the reverse-DNS style naming. 

Group names are often domain style, which means for a group name `bar.k8s.io` you want your type `Foo` for version `v1` to be named:

`io.k8s.bar.v1.Foo` - In reverse DNS style. As it was before this patch.

or `Foo.v1.bar.k8s.io` - In forward DNS style which is an option.

`bar.k8s.io.v1.Foo` does not make as much sense.

I say we opt back for reverse-DNS since it is the current behavior, is familiar to a class of developers (I believe this was original reasoning for reverse-DNS: See https://github.com/kubernetes/kubernetes/pull/40124#discussion_r97702003).

Reply:
I've reverted to using the original reverse-DNS style names.

Reply:
am not super opionated. Either is fine.

Reply:
No strong opinions either, as long as no naming conflict is introduced. reverse-DNS should be fine